### DISCUSSION TITLE
Mh/s rates

### DISCUSSION THREAD

**Discussion by wildgrizzly1** [Lolliedieb/lolMiner-releases#144] [Q&A]
Hello,

Recently started running lolminer, but I have a question about running multiple GPUs in the same system. Using GTX 1070, I get about 26 Mh/s. When a second GTX 1070 GPU is installed, I still the get same approximate 26 Mh/s instead of doubling up to about 52 Mh/s. The miner reports each GPU running at about 13 Mh/s. Any suggestions would be greatly appreciated, thank you.

**Comment by Westy69** [comment]
Try Using T-Rex Miner instead.
Lol Miner is not set up for Nvidia Chipsets as far as I can tell.

**Reply by stevepresley** [reply]
@Westy69 lolMiner is faster than trex on my 1660 Tis.. check out #158 for a solution

**Comment by wildgrizzly1** [comment]
Hello Westy69, thanks for the reply. I switched to phoenix and things seem to work as expected... thank you.

**Reply by Westy69** [reply]
Glad it helped.
